manubot/process/bibliography.py

```diff
@@ -24,12 +24,12 @@ def load_bibliography(path):
     try:
         if path.suffix == '.json':
             use_pandoc_citeproc = False
-            with path.open() as read_file:
+            with path.open(encoding='utf-8-sig') as read_file:
                 csl_items = json.load(read_file)
         if path.suffix == '.yaml':
             use_pandoc_citeproc = False
             import yaml
-            with path.open() as read_file:
+            with path.open(encoding='utf-8-sig') as read_file:
```

Comment:
Possible that `utf-8` alone would be sufficient because I believe the BOM is considered whitespace. Actually, possibly not according to https://stackoverflow.com/a/13156715/4651668.